api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -202,5 +202,7 @@
 	
 	public static final String TAG_SEARCH_HANDLER = "tag.search.handler";
 	
-	public static final String HAPI_DEFAULT_NARRATIVES_PROPERTY_FILE = "classpath:ca/uhn/fhir/narrative/narratives.properties";
+	public static final String HAPI_NARRATIVES_PROPERTY_FILE = "classpath:ca/uhn/fhir/narrative/narratives.properties";
+	
+	public static final String OPENMRS_NARRATIVES_PROPERTY_FILE = "file:src/main/resources/narratives.properties";
```

Comment:
This should be "classpath:narrative.properties"

Reply:
@ibacher If I do that, it uses the `narratives.properties` of HAPI, instead of the local one. So I initially kept it as `defaultNarratives.properties`, but later I changed it to `narratives.properties`. So I used the file path instead. So should I rename it back?

Reply:
@ibacher If I do that, it uses the narratives.properties of HAPI, instead of the local one. So I've moved the resources to a sub-directory `org/openmrs/module/fhir2/` inside the `resources` directory, and changed `OPENMRS_NARRATIVES_PROPERTY_FILE` to `classpath:org/openmrs/module/fhir2/narratives.properties`.

Reply:
Ok. My point was primarily that `file:` with `src/main/resources/...` is only going to work in a development environment. Instead of differentiating these two, it might make sense to just use something like `classpath*:narratives.properties`, allowing us to use other `narratives.properties` files that exist on the classpath. Basically, we want to be forced to know as little as possible about HAPI.

Reply:
Ah, I see.